tests/integration/test_airbase.py

```diff
@@ -10,27 +10,28 @@
 
 
 @pytest.fixture(scope="module")
-def withoutresponses():
-    with rsps.RequestsMock(passthru_prefixes="http"):
-        yield
-
-
-@pytest.fixture(scope="module")
-def client(withoutresponses):
+def client():
     """Return an initialized AirbaseClient"""
-    return airbase.AirbaseClient(connect=True)
+    prefixes = [
+        airbase.resources.E1A_SUMMARY_URL,
+        "http://fme.discomap.eea.europa.eu/",
+        "https://ereporting.blob.core.windows.net/",
+        airbase.resources.METADATA_URL,
+    ]
+    with aioresponses(passthrough=prefixes):
+        yield airbase.AirbaseClient(connect=True)
```

Comment:
Is there much utility to this since it covers all the calls that the client could make?

Reply:
When I was developing the fetch helpers, the integration tests failed because because they could not reach the EEA servers.
At the time, I was using the `pytest-aioresponses` plugin then, so it might no longer be the case now that I'm using `aioreponses` directly.
Will check and come write again.

Reply:
It was no longer needed, so I removed it

Reply:
Ah gotcha. Yeah the integration tests actually do real calls to try to catch possible changes in formatting etc. So those ones shouldn't be mocked out.